### PR TITLE
deps: bump dependency versions

### PR DESCRIPTION
**Pull request by stephaniewang526**
Fixes #31 #51 #52

### CODE REVIEW

File: google-cloud-bigquerystorage/pom.xml [hunk 1]
```diff
@@ -105,7 +105,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-bigquery</artifactId>
-      <version>1.102.0</version>
+      <version>1.106.0</version>
```
Comment (elharo) [inline google-cloud-bigquerystorage/pom.xml]: This is likely the upgrade that's causing the issues. I'd try splitting this out into a separate PR to isolate it.

File: pom.xml [hunk 2]
```diff
@@ -108,6 +110,16 @@
         <artifactId>grpc-google-cloud-bigquerystorage-v1beta2</artifactId>
         <version>0.85.2-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-bigquerystorage-v1beta2:current} -->
       </dependency>
+      <dependency>
```
Comment (elharo) [inline pom.xml]: I don't think you need to add these two. Just move avro to the end of your dependencies section.

Comment (stephaniewang526) [reply pom.xml]: will do

Comment (elharo) [reply pom.xml]: Or maybe not. The avro dependency on jackson is not as deep.

**Pull request merged**